### PR TITLE
Alternative-Dark theme

### PR DESCRIPTION
**Pull request by fabianski7**
just a new theme 
https://i.imgur.com/oPHlHJ9.png

### CODE REVIEW

**Review by Frenzie — COMMENTED** [review]
Should be fine, besides the obvious:

```
  666:9   ✖  Expected "#ABABAB" to be "#ababab"     color-hex-case   

  882:5   ✖  Expected indentation of 1 tab          indentation      

 1219:9   ✖  Expected indentation of 2 tabs         indentation      

 1219:21  ✖  Expected "#333333" to be "#333"        color-hex-length 

 1315:1   ✖  Unexpected whitespace at end of line   no-eol-whitespace
```

**Comment by fabianski7** [comment]
serious? change #BFBFBF to #bfbfbf?

**Comment by Alkarex** [comment]
Hello @fabianski7 ,
Thanks for this contribution :-)
Could you please fix the minor warnings shown in https://travis-ci.org/FreshRSS/FreshRSS/jobs/656578554?utm_medium=notification&utm_source=github_status ?

**Comment by fabianski7** [comment]
what are you using to edit this CSS file? vscode?

I'm using the sublime and [I don't see any problems ](https://upload.vaa.red/G2w4H#ATqMWPK3Pxcsr2fw0Apg0HmUutVrErvU)with the formatting

**Comment by Alkarex** [comment]
@fabianski7 It is really minor, and automatically caught by our Travis continuous integration, which runs various tests, including for basic coding style.
See e.g. https://travis-ci.org/github/FreshRSS/FreshRSS/jobs/656578554?utm_medium=notification&utm_source=github_status for details.
This is an example of what you had before, with wrong indenting as well as a mix of tabs and spaces:
image
I believe there is an option, also in Sublime, to display white spaces.
Everything should be fixed now :-)

**Comment by Alkarex** [comment]
P.S. You can also instruct your text editor to use https://github.com/FreshRSS/FreshRSS/blob/master/.editorconfig and automatically pick the correct white-space convention in a given project
P.P.S. We do not have a rule enforced in `.editorconfig` for CSS files, but it just needs to be consistent

**Comment by Frenzie** [comment]
That looks like a Scintilla-based editor, but I figure it's not Geany. Notepad++ perhaps?

(The question + screenshot made me curious. I use Geany most of the time. :-P)

**Comment by Alkarex** [comment]
@Frenzie Notepad++ indeed on my side for this project, when I am on Windows :-P

**Comment by Frenzie** [comment]
Notepad++ is my preferred Windows editor too. ^_^

**Comment by Alkarex** [comment]
This all looks good @fabianski7 , thanks again :-)

**Comment by fabianski7** [comment]
I didn't know about the `.editorconf` file, thanks for the tip.
But I guess there's no rule for spaces in `*.css` files just `*.php` no?

```
[*]
end_of_line = lf
insert_final_newline = true
charset = utf-8

[*.php]
indent_style = tab
indent_size = 4
insert_final_newline = true
```

**Comment by Alkarex** [comment]
@fabianski7 Indeed. So for CSS, it just needs to be consistent

File: p/themes/Alternative-Dark/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Alternative Dark",
+  "author": "Ghost",
```
Comment (math-GH) [inline p/themes/Alternative-Dark/metadata.json:3]: @fabianski7
is there a reason why you chose "Ghost" as author name and not your username?

Comment (fabianski7) [reply p/themes/Alternative-Dark/metadata.json:3]: I honestly don't remember 😅. I don't care much for credits

**Pull request merged**